vignettes/detailed_example.Rmd

```diff
@@ -49,7 +51,7 @@ parent <- data.frame(uuid=1:10, age=sample(30:60,10,replace = T),parent_gender=s
 child <- data.frame(parent_uuid=sample(1:10,20,replace = T), age=sample(20,1,18), child_gender=sample(c("F","M"),20,replace = T))
 ```
 
-## Affect parent age to each child
+## Attach parent age to each child
```

Comment:
this should not change unless function names are changed as well

Reply:
reverted this back! thanks